lib/galaxy/tools/__init__.py

```diff
@@ -1989,6 +1990,7 @@ def to_json(self, trans, kwd={}, job=None, workflow_building_mode=False):
             'id'            : self.id,
             'help'          : tool_help,
             'citations'     : bool(self.citations),
+            'xrefs'         : bool(self.xrefs),
```

Comment:
What about adding:
```python
        tool_dict["xrefs"] = self.xrefs
```
around line 1880 and remove this line?

Reply:
Ok, but I am not sure I understand the usage of the tool_model